### DISCUSSION TITLE
Set Login Page as initial without breaking dashboard page

### DISCUSSION THREAD

**Discussion by BrianWakhutu** [primefaces/community#707] [PrimeFaces Templates] [answered]
Could someone show me how to set up the login page as the initial page without breaking the dashboard page? I am using the Ultima template

**Comment by mertsincan** [comment, marked as answer]
Hi,

You can set <welcome-file-list> in web.xml;
https://stackoverflow.com/questions/9295878/how-to-display-a-default-home-page

in Ultima
```
<welcome-file-list>
        <welcome-file>favorites/dashboard.xhtml</welcome-file>
    </welcome-file-list>
```
